base/bin/migration_template.mustache

```diff
@@ -0,0 +1,10 @@
+const r = require('root/lib/db');
```

Comment:
Can't do much migrating without a database connection! That's so that someone can come in and write something like:

```
r.table('something').filter({some: 'thing'}).run().then(data => doMigrationStuff(data));
```